source/NVDAObjects/IAccessible/adobeAcrobat.py

```diff
@@ -137,15 +143,30 @@ def _getNodeMathMl(self, node):
 		yield "</%s>" % tag
 
 	def _get_mathMl(self):
+		if self.pdDomNode is None:
+			log.debugWarning("_get_mathMl: self.pdDomNode is None!")
+			raise LookupError
 		# There could be other stuff before the math element. Ug.
+		mathMl = self.pdDomNode.GetValue()
+		log.debug(
+			f'\n_get_mathMl: math recognized: {mathMl.startswith("<math")}, child count={self.pdDomNode.GetChildCount()}',
+		)
+		log.debug(f"\nname={self.pdDomNode.GetName()}\nvalue={self.pdDomNode.GetValue()}")
+		# this test and the replacement doesn't work if someone uses a namespace tag (which they shouldn't, but..)
+		if mathMl.startswith("<math"):
+			return mathMl.replace('xmlns:mml="http://www.w3.org/1998/Math/MathML"', "")
```

Comment:
What happens if this is `False`?

Reply:
If `mathMl.startswith("<math")` is false, it moves on to see if there is "fully tagged" MathML. 

Fully tagged PDF math is where each subexpression is tagged with a child MathML element (e.g., `mfrac`) and points to the subpart of the rendered expression. It turns out this is very hard to generate, so there are very few documents in the world that do that. It is however legal and mentioned in the PDF/UA-2 spec. If it doesn't find any children, there may be alt text as the content. That is what the last line does. If there isn't content, then it is an empty `<mtext>` element, which is legal.

With FoxIt Reader, it there is MathML in an associated file, the test will be true. Since Adobe Reader doesn't yet deal with associated files, the test will be false, it won't find any children, and alt text will be returned by the last line.

Reply:
Thanks for the thorough explanation @NSoiffer. Also, apologies for the rather silly question. I was out of it yesterday, and somehow missed the `return`.